Assets/Scripts/Boat/BoatMovement.cs

```diff
@@ -33,22 +42,39 @@ public class BoatMovement : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
-        
+        SetVerticalBoundsBasedOnScreenSize();
+    }
+
+    private void SetVerticalBoundsBasedOnScreenSize()
```

Comment:
What about when the screen gets resized after Start() is called?

Reply:
Unaccounted for. Ill add that in now.